pkg/cmd/pulumi/package_gen_sdk.go

```diff
@@ -49,53 +54,90 @@ func newGenSdkCommand() *cobra.Command {
 			}
 			if language == "all" {
 				for _, lang := range []string{"dotnet", "go", "java", "nodejs", "python"} {
-					err := genSDK(lang, out, pkg)
+					// If we're generating all the languages place each one in a subdirectory of the output
+					// directory, and grab the overlays from a subdirectory of the overlays directory.
+					langOut := filepath.Join(out, language)
+					langOverlays := ""
+					if overlays != "" {
+						langOverlays = filepath.Join(overlays, language)
+					}
+					err := genSDK(lang, langOut, pkg, langOverlays)
 					if err != nil {
 						return err
 					}
 				}
 				return nil
 			}
-			return genSDK(language, out, pkg)
+			return genSDK(language, out, pkg, overlays)
```

Comment:
Intentional, no one's using this yet and I kept running into issues in experimenting with other uses of this where not being able to control the full path was limiting.

Reply:
As a potential consumer 👍

Reply:
We ended up reverting this part because it turned out a load of downstream was already using the expectation of suffixing language name :( 
I'd like to change this but we'll need to work out how to sync it across to pulumi-command and the bigger issue, the boilerplate repo.